src/ucp/tag/rndv.c

```diff
@@ -1209,6 +1218,7 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_rtr_handler,
         /* Do not deregister memory here, because am zcopy rndv may
          * need it registered (if am and tag is the same lane). */
         ucp_tag_offload_cancel_rndv(sreq);
+        sreq->flags &= ~UCP_REQUEST_FLAG_OFFLOADED;
```

Comment:
maybe move it to `ucp_tag_offload_cancel_rndv`?

Reply:
fixed